src/frontend/src/pages/HomePage/RejectPage.tsx

```diff
@@ -0,0 +1,31 @@
+import { Typography, Box, Grid } from '@mui/material';
```

Comment:
You have a linting issue which means that you have imported grid even though you are not using it, you should get rid of the import here and we should be good to merge